src/components/FrameworkSwitcher.tsx

```diff
@@ -76,6 +76,11 @@ const FrameworkSwitcher = forwardRef(
       const val = e.target.value;
       if (!storeFramework(val)) return;
 
+      // Update query param for sharing selection
+      const url = new URL(window.location.toString());
+      url.searchParams.set("f", val);
+      history.pushState({}, "", url);
```

Comment:
In testing this out on the website, I think it might make more sense to be `replaceState`. Should we get some others' opinion?

Reply:
I would probably replace in the case the previous state was invalid, but not being this the case and since switching is a user decision it feels natural to provide a back action.

cc @david

Reply:
Yeah, not breaking the back button (in fact, treating the framework switch as a navigation event), is my preference.